### PR TITLE
WIP - add more icons.

### PR DESCRIPTION
**Pull request by bwinton**
I copied the icons from the google drive, and lightly editted the json file. More help adding new entries would be greatly appreciated! :wink:

### CODE REVIEW

**Comment by bwinton** [comment]
Oh, yeah, we should add an `svgo` step in the build script. I'll do that in this patch.

You can see the changes by editing your `package.json` and pointing the `photon-icons` version to the direcetory on your filesystem where the code lives (like `../photon-icons/`)

**Comment by bwinton** [comment]
(Oh, you'll need to add the extra icons to the `photon-icons.json` first. I guess I could take a stab at that…)

**Comment by ExE-Boss** [comment]
Shouldn’t the icons keep the unminimised license headers?

**Comment by bwinton** [comment]
Enh. I dunno. It's hard to get svgo to not minimize the comments, and I feel like the license is still valid in this form, so... 🤷‍♂️

**Comment by aminalhazwani** [comment]
@bwinton I did a first content editing, we can merge this and do a second iteration after I file a couple of issues for following things.

**Comment by bwinton** [comment]
One more thing... https://www.mozilla.org/en-US/MPL/2.0/FAQ/#full-copy-only suggests that we might not need the header on every file, so maybe we should remove it, and add it back when people click the download button?

**Comment by ExE-Boss** [comment]
Or, add it during the publishing to `npm` process, that way, once I make use of this in my [Application Menu extension](https://github.com/ExE-Boss/app-menu) [(Mozilla Add‑on)](https://addons.mozilla.org/firefox/addon/application-menu/) for [the Photon theme](https://github.com/ExE-Boss/app-menu/tree/master/src/themes/photon), I don’t have to add it myself to every file (since my Application Menu extension is licensed under the GPL-3.0).

**Comment by aminalhazwani** [comment]
I am soon to add also some Android icons, do not merge 🙂

**Comment by aminalhazwani** [comment]
@bwinton I added a couple of Android icons and started to fill some `tags` values. Question: when I run `npm run test` I get this error

```
Poorly-sized entries:
  icons/android/info-24.svg does not have a proper viewBox.
  icons/android/back-24.svg does not have a proper viewBox.
  icons/android/forward-24.svg does not have a proper viewBox.
  icons/android/refresh-24.svg does not have a proper viewBox.
  icons/android/stop-24.svg does not have a proper viewBox.
  icons/android/data-collection-24.svg does not have a proper viewBox.
  icons/android/delete-24.svg does not have a proper viewBox.
  icons/android/pin-24.svg does not have a proper viewBox.
  icons/android/pin-outline-24.svg does not have a proper viewBox.
  icons/android/preferences-24.svg does not have a proper viewBox.
  icons/android/turbo-mode-24.svg does not have a proper viewBox.
  icons/android/turbo-mode-outline-24.svg does not have a proper viewBox.
  icons/android/top-sites-24.svg does not have a proper viewBox.
  icons/android/secure-24.svg does not have a proper viewBox.
```
even if those files have a `viewBox`, any idea why?

File: icons/android/back-24.svg [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. --><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M21 11H5.914l5.793-5.793a1 1 0 1 0-1.414-1.414l-7.5 7.5a1 1 0 0 0 0 1.414l7.5 7.5a1 1 0 0 0 1.414-1.414L5.914 13H21a1 1 0 0 0 0-2z"/></svg>
\ No newline at end of file
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. --><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.707 3.293a1 1 0 0 1 0 1.414L5.414 11H21a1 1 0 1 1 0 2H5.414l6.293 6.293a1 1 0 0 1-1.414 1.414l-8-8a1 1 0 0 1 0-1.414l8-8a1 1 0 0 1 1.414 0z" fill="#000"/></svg>
```
Comment (ExE-Boss) [inline icons/android/back-24.svg]: Last I checked, `fill` should be set to `context-fill` and not `none` or `#000000`, or not set at all.

Comment (aminalhazwani) [reply icons/android/back-24.svg]: @ExE-Boss iirc there is no `context-fill` on Android nor iOS, it's a Firefox for Desktop only feature.

The fill set to `none` is not necessary and it should be removed, thanks!

Comment (ExE-Boss) [reply icons/android/back-24.svg]: `context-fill` is an [official SVG 2.0 feature](https://svgwg.org/svg2-draft/painting.html#TermContextElement), Firefox just has the [`-moz-context-properties` property](https://developer.mozilla.org/docs/Web/CSS/-moz-context-properties) to allow setting it on SVG images which are embedded in an HTML `` tag (see also https://github.com/w3c/svgwg/issues/348).

Comment (aminalhazwani) [reply icons/android/back-24.svg]: Yeah, thanks for the clarification ☺️  I meant that we don't use `context-fill` on mobile. Let me talk more with mobile engineers in order to understand how they can set the color of an icon if they import all the icons with just one package.

**Comment by aminalhazwani** [comment]
Ok, I changed the fill to Grey 90 a80, so an output XML is for example:

```
<?xml version="1.0" encoding="utf-8"?>
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
    <path
        android:fillColor="#0C0C0D"
        android:fillAlpha=".8"
        android:fillType="evenOdd"
        android:pathData="M13 4a1 1 0 1 0-2 0v7H4a1 1 0 1 0 0 2h7v7a1 1 0 1 0 2 0v-7h7a1 1 0 1 0 0-2h-7V4z"/>
</vector>
```
I am going to test the XMLs and add the new icons to the JSON. When I `npm run test` I get this error, any idea @bwinton?

```
Poorly-sized entries:
  icons/android/info-24.svg does not have a proper viewBox.
  icons/android/back-24.svg does not have a proper viewBox.
  icons/android/forward-24.svg does not have a proper viewBox.
  icons/android/refresh-24.svg does not have a proper viewBox.
  icons/android/stop-24.svg does not have a proper viewBox.
  icons/android/data-collection-24.svg does not have a proper viewBox.
  icons/android/delete-24.svg does not have a proper viewBox.
  icons/android/pin-24.svg does not have a proper viewBox.
  icons/android/pin-outline-24.svg does not have a proper viewBox.
  icons/android/preferences-24.svg does not have a proper viewBox.
  icons/android/turbo-mode-24.svg does not have a proper viewBox.
  icons/android/turbo-mode-outline-24.svg does not have a proper viewBox.
  icons/android/top-sites-24.svg does not have a proper viewBox.
  icons/android/secure-24.svg does not have a proper viewBox.
```

**Comment by bwinton** [comment]
Yeah, the new SVGs have to start with `<svg xmlns="http:\/\/www.w3.org\/2000\/svg" width="<something>" height="<same thing>" viewBox="0 0 <same thing> <same thing>"` and then whatever other attributes you want.

**Comment by aminalhazwani** [comment]
Ok, gonna check this!

**Comment by aminalhazwani** [comment]
Uhm, I am not sure if that is the issue, if I compare `add-24.svg` that doesn't prompt any error 

```
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M13 4a1 1 0 1 0-2 0v7H4a1 1 0 1 0 0 2h7v7a1 1 0 1 0 2 0v-7h7a1 1 0 1 0 0-2h-7V4z" fill="#0C0C0D" fill-opacity=".8"/></svg>
```

with `back-24.svg` that is listed as poorly-sized entry I cannot see the difference:

```
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.707 3.293a1 1 0 0 1 0 1.414L5.414 11H21a1 1 0 1 1 0 2H5.414l6.293 6.293a1 1 0 0 1-1.414 1.414l-8-8a1 1 0 0 1 0-1.414l8-8a1 1 0 0 1 1.414 0z" fill="#0C0C0D" fill-opacity=".8"/></svg>
```

**Pull request merged**